### PR TITLE
Bug fixes/issue84

### PR DESCRIPTION
**Pull request by OsirisTerje**
Change detection and message of NUnit 3 tests.

### CODE REVIEW

File: src/NUnitTestAdapter/TestLogger.cs [hunk 1]
```diff
@@ -60,7 +60,7 @@ public void LoadingAssemblyFailedWarning(string dependentAssembly, string source
 
         public void NUnitLoadError(string sourceAssembly)
         {
-            SendErrorMessage("NUnit failed to load " + sourceAssembly);
+            SendInformationalMessage("No NUnit 2 tests found in " + sourceAssembly);
```
Comment (CharliePoole) [inline src/NUnitTestAdapter/TestLogger.cs]: See my comment on the 3.0 issue. Seems like a method named NUnitLoadError should issue error messages... either we should not use it or it should have another name.

**Comment by CharliePoole** [comment]
@OsirisTerje What's the status of this PR?

**Comment by OsirisTerje** [comment]
Imho:  We should rename the NUnitLoadError method, and then merge.  I'll do it now.

**Comment by CharliePoole** [comment]
Sounds good to me!

**Comment by OsirisTerje** [comment]
@rprouse The build fails on the MS object model, I assume that is what you have fixed in the other branch, right?  Should we merge this - or fix the same thing here first?  Right now the reference is correct for release, but then fails for running tests.

**Comment by rprouse** [comment]
@OsirisTerje we created a private NuGet package that contains the MS Object Model, uploaded it to MyGet and then referenced it instead of the versions in the GAC. @CharliePoole made those changes in PR #122, so if we are ready to merge that, you can then merge latest from master into this PR to get the build working.

**Pull request merged**